Articles/WhatIs/WhatIsUserExperienceDesign.md

```diff
@@ -0,0 +1,20 @@
+### What is User Experience Design?
+#### Contributed by  [David E. Bernholdt](https://github.com/bernhold)
+#### Publication date: February 20, 2024
+
+<!--deck start--->
+User experience refers to how the user of a piece of software (or other system) interacts with and experiences it.  A thoughtfully designed user experience can make a huge difference to the reception of a piece of software.
+<!--deck end--->
+
+<!--body start--->
+According to the international standard on the ergonomics of human-system interactions (ISO 9341), user experience (UX) is a "user's perceptions and responses that result from the use and/or anticipated use of a system, product, or service."  It is often considered to encompass usability, but also more than that.
```

Comment:
Is the `a` in `...(UX) is a "user's perceptions...` right? I don't think it is.

Reply:
That's a quote from the ISO standard.

Reply:
> That's a quote from the ISO standard.

I am sure it is an important quote but, as written, it didn't parse for me for several mins.

Reply:
Isn't the ISO doc number 9`2`41? 

Here is a proposed revision that I think is clearer.

```suggestion
The international standard on the ergonomics of human-system interactions ([ISO 9241](https://en.wikipedia.org/wiki/ISO_9241)), defines *user experience (UX)* as the "user's perceptions and responses that result from the use and/or anticipated use of a system, product, or service."
It is often considered to encompass usability, but also more than that.
```